container/check_container.go

```diff
@@ -95,6 +64,51 @@ func (c *containerCheck) Run(ctx context.Context) (certification.Results, error)
 	return eng.Results(ctx), nil
 }
 
+func (c *containerCheck) resolve(ctx context.Context) error {
+	if !c.resolved {
```

Comment:
Same as other block. Please swap this around to keep the indentation to a minimum.

Reply:
fixed